client/blocks/site/style.scss

```diff
@@ -116,7 +116,7 @@
 	overflow: hidden;
 	white-space: nowrap;
 	&::after {
-		@include long-content-fade();
+		@include long-content-fade( $color: var( --sidebar-secondary-background-gradient ) );
```

Comment:
So this sass mixin does work with css custom properties? I wasn't sure about if it's possible.

Reply:
Yes, I updated the mixin to work with CSS custom properties. You just need to make sure that the color value assigned to the property you pass in is wrapped in `hex-to-rgb()` function.

Reply:
Updated the documentation for the Mixin as well 👍